ads/google/imaVideo.js

```diff
@@ -1133,4 +1137,35 @@ const VideoEvents = {
    * @event reload
    */
   RELOAD: 'reloaded',
+  /**
+   * ended
+   *
+   * Fired when the video ends.
+   *
+   * @event ended
+   */
+  ENDED: 'ended',
```

Comment:
nit: please change line 1117 to say `@event unmuted` rather than `@event pause` (unrelated)